cgi/search.pl

```diff
@@ -654,11 +654,21 @@
 
 	my $share = lang('share');
 
-
+	my $map = '';
```

Comment:
This works, but you can do it in one line instead of 4 like this:

my $map = param("generate_map") || '';

If the first expression is true (it is defined and the value is not false), then it return the first expression, otherwise it returns the right expression.

Reply:
How's this?